wrappers/azurekeyvault/azurekeyvault.go

```diff
@@ -365,3 +387,31 @@ func ParseKeyVersion(kid string) string {
 	keyVersionParts := strings.Split(kid, "/")
 	return keyVersionParts[len(keyVersionParts)-1]
 }
+
+func cloudConfigFromName(name string) (cloud.Configuration, error) {
+	configs := map[string]cloud.Configuration{
+		azureChinaCloudEnvName:  cloud.AzureChina,
+		azurePublicCloudEnvName: cloud.AzurePublic,
+		azureUSGovCloudEnvName:  cloud.AzureGovernment,
+	}
+	name = strings.ToUpper(name)
+	c, ok := configs[name]
+	if !ok {
+		return c, fmt.Errorf("err: no cloud configuration matching the name %q", name)
+	}
+	return c, nil
+}
+
+func keyVaultDNSSuffixFromName(envName string) (string, error) {
+	urls := map[string]string{
```

Comment:
Same here?

Alternately it could be one function where the env names map to structs containing both the cloud config and dns suffix and the calling function can just choose which it wants from the returned struct.

Reply:
Yeah definitely can do that, thanks for the suggestion!

Reply:
Fixed in 4d94f4c9288da3d640e9bcd16137c337382b650e